valhalla/thor/costmatrix.h

```diff
@@ -333,20 +333,6 @@ class CostMatrix : public MatrixAlgorithm {
     return infos;
   };
 
-  void ModifyHierarchyLimits() {
-    // Distance threshold optimized for unidirectional search. For bidirectional case
```

Comment:
`expand_within_dist` wasn't used anyway in CostMatrix, it was tested but the performance drain turned out to be too big.